### ISSUE TITLE
Create API variants for websockets that accept Source or Sink

### ISSUE THREAD

**Issue by drewhk** [akka/akka-http#445]
Many people want just one-directional communcation, so they have a Sink or Source, we should give them helper APIs to be able to use them without hassle.

**Comment by ktoso** [comment]
I agree, it should be a directive like `handleWebsocketWith(Source, incomingDataMode???)`
The second param optional and should default to Sink.Ignore (or perhaps 2nd param not needed),
same for the Sink.

Perhaps it could install a PING keep-alive (that's why I'm thinking about the modes)?

**Comment by atemerev** [comment]
What's wrong with plugging the other end with Sink.ignore? Perhaps, the documentation could be improved on this.

**Comment by drewhk** [comment]
There isn't anything wrong, it is just a common use-case and people are confused all the time. Docs are good, but in this particular case I would not mind a little bit of API sugar since people might see the methods in autocomplete but not reach for the docs.

**Comment by jrudolph** [comment]
The problem with `Sink.ignore` is that it won't consume the streamed entities which might stall slow streams on receipt of very long frames.
